creature/player/player.h

```diff
@@ -10,7 +10,7 @@ class Player : public Creature {
     void action(sf::Event& event, float time, Field* game_field,
                 const std::vector<std::shared_ptr<Creature>>& drawable_creatures);
 
-    Inventory inventory;
+    static inline Inventory inventory;
     float get_satiety() {
```

Comment:
Why this field is static?

Reply:
we can discuss it in diskord, babe

Reply:
Do you DEEPly convinced it that?

Reply:
unfortunately no;( there are some problems, I'd like to talk about them